.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - 'v[0-9]+.[0-9]+.[0-9]+' # Push events on version tag
+  pull_request:
+    branches:
+      - master
+
+jobs:
+
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9"]
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -U importlib_metadata
+        pip install -r requirements.txt
+    - name: Run tests
+      run: make test
+
+  deploy:
+    name: Deploy to PyPI
+    runs-on: ubuntu-20.04
+    needs: test
+    permissions:
+      # IMPORTANT: this permission is mandatory for trusted publishing
+      # https://docs.pypi.org/trusted-publishers/using-a-publisher/
+      id-token: write
+    if: startsWith(github.ref, 'refs/tags/v')
```

Comment:
Maybe this needs another check to avoid triggering on personal forks?

Reply:
Sorry, missed your comment.
Anyway, I think the since the flow is triggered on a branch of on a tag with regex `v[0-9]+.[0-9]+.[0-9]+` and I check here the tag it should be safe (the travis triggers we had were also on `tag =~ ^v`

I'll create a PR with branch starting with v to be sure I'm not missing anything.

Or did I get your concern right?

Reply:
The potential issue is when I create a tag starting with 'v' on itamaro/ybt - that shouldn't trigger a release.
Probably not a big deal though, I'm guessing it would fail because of permissions.

Reply:
OK.
See https://github.com/resonai/ybt/pull/256 
I guess that after merging it I'll try to release and you can try to see if there are false-positive (reasonable ones :-) )